### PR TITLE
Calculate Annotation Sufficiency

### LINKED ISSUE #33: Add annotation sufficiency
We should provide realtime feedback to patients about how diagnostically useful their profile is.
screen shot 2018-06-07 at 1 00 14 pm.
Just use a canned library for password strength and adapt it.

@kshefchek will comment about how to wire up the API.

### PR DESCRIPTION
**Pull request by heathharrelson**
Implement the annotation sufficiency calculation. Related to #33, with UI work to come in a subsequent PR.

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -32,11 +36,39 @@ export default {
...
+    /**
+     * Sets the quality score to the specified value.
+     * @param {Object} state - the current state.
+     * @param {Number} score - the quality score to set.
+     */
+    setQualityScore(state, score) {
+      state.qualityScore = score;
+    }
+  },
+
```
Comment (heathharrelson) [inline src/store/index.js]: Whereas mutations are synchronous and update the store directly, actions may be asynchronous and update the store via committing a mutation.

**Comment by heathharrelson** [comment]
[The Vuex documentation](https://vuex.vuejs.org/) may be helpful in understanding this PR.

File: test/unit/specs/components/ResultsPage.spec.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { shallow, createLocalVue } from '@vue/test-utils';
 import ResultsPage from '@/components/ResultsPage';
 import PdfRenderer from '@/utils/pdf-renderer';
 import isExportDocument from '@/utils/is-export-document';
-import exampleResponses from '../../example-responses';
+import exampleTerms from '../../example-terms';
```
Comment (aeyates) [inline test/unit/specs/components/ResultsPage.spec.js]: I notice that you renamed this file but did not change the const declaration and export. It still reads exampleResponses and should probably be renamed for consistency.

Comment (heathharrelson) [reply test/unit/specs/components/ResultsPage.spec.js]: Good point. I have renamed the constant to match.

**Pull request merged**